lib/livebook/hubs/dockerfile.ex

```diff
@@ -402,7 +419,7 @@ defmodule Livebook.Hubs.Dockerfile do
 
         "team" ->
           [
-            if app_settings.access_type == :public and config.zta_provider != :livebook_teams do
+            if app_settings.access_type == :public do
```

Comment:
We need to check if the auth is enabled, no?

Reply:
Ah it's airgapped. So what we actually want is change the warning message.

Reply:
We changed this recently, but this one should actually still say something like `configure a password or configure LIVEBOOK_IDENTITY_PROVIDER environment variable.`